solana/solana-ibc/programs/solana-ibc/src/storage/ids.rs

```diff
@@ -160,8 +160,10 @@ impl PortChannelPK {
         })
     }
 
+    #[allow(dead_code)]
     pub fn port_id(&self) -> ibc::PortId { ibc::PortId::from(&self.port_key) }
 
+    #[allow(dead_code)]
```

Comment:
```suggestion
    pub fn port_id(&self) -> ibc::PortId { ibc::PortId::from(&self.port_key) }

```
I don’t think those are necessary either.